lib/client/js/ui/base/Control.js

```diff
@@ -135,6 +135,10 @@ var Control = (function() {
         return this._$;
     };
 
+    Control.prototype.attachEvent = function() {
```

Comment:
Nit: we should rename this method to `attachEvents` since there might be a bunch of them attached.